pkg/services/sqlstore/active_node.go

```diff
@@ -7,11 +7,15 @@ import (
 
 	"github.com/grafana/grafana/pkg/bus"
 	m "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/setting"
 )
 
 var (
-	insertHeartbeatSQL = "insert into active_node(node_id, heartbeat, part_id, alert_run_type, alert_status) values(?, ?, ?, ?, ?)"
-	getNextPartIDSQL   = "select count(part_id) as part_id from active_node where heartbeat = ? and alert_status = ?"
+	insertHeartbeatSQL  = "insert into active_node(node_id, heartbeat, part_id, alert_run_type, alert_status) values(?, ?, ?, ?, ?)"
+	getNextPartIDSQL    = "select count(part_id) as part_id from active_node where heartbeat = ? and alert_status = ?"
+	lastCleanupCheckSQL = "select * from active_node as a where a.heartbeat > ? and a.heartbeat <= ? and alert_run_type='" + m.CLN_ALERT_RUN_TYPE_CLEANUP + "'"
```

Comment:
What is the need to check alert_run_type in where clause?. Why not just check the heartbeat?